clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -164,6 +170,30 @@ private void parseContainer(XMLExtendedStreamReader reader, PathAddress subsyste
                     this.parseTransport(reader, address, operations);
                     break;
                 }
+                case EVICTION_THREAD_POOL: {
+                    if (this.schema.since(InfinispanSchema.VERSION_3_0)) {
```

Comment:
Dwprecated attributes are meant to cascade to the default action, thus the switch cases need to be ordered with care.